_accounts.md

```diff
@@ -0,0 +1,316 @@
+# Accounts
+Uphold allows users to deposit value into a specific card from an external source (ACH account, debit/credit card or wire transfer) or withdraw to an external source (ACH account or wire transfer).
+
+Whenever a deposit is made into an Uphold card, it will be automatically converted into the value determined by the card's denomination. Likewise, when a withdrawal is made, the currency will be converted to the currency of the destination account, thus minimizing fees and currency conversions.
+
+We support the following account types:
+
+Account type | Deposits supported? | Withdrawals supported?
+------------ | ------------------- | ----------------------
+ach          | Yes                 | Yes
+card         | Yes                 | No
+sepa         | Yes                 | Yes
+
+Please refer to our FAQ for estimated [ACH transaction times](https://support.uphold.com/hc/en-us/articles/206762103-How-to-add-and-withdraw-funds-via-bank-transfer-U-S-), [SEPA transaction times](https://support.uphold.com/hc/en-us/articles/205803186-How-to-add-and-withdraw-funds-via-bank-transfer-Europe-), [fees and limits](https://support.uphold.com/hc/en-us/articles/206118653-Transaction-Trading-Limits).
+
+## List Accounts
```

Comment:
Would be useful to specify which OAuth scope is required in order to make this call.